client/src/components/Navbar.js

```diff
@@ -1,16 +1,80 @@
-import React from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import Signup from './Signup';
+import AuthButtons from './AuthButtons';
+
+const styles = () => ({
+  root: {
+    flexGrow: 1,
+  },
+  grow: {
+    flexGrow: 1,
+  },
+});
+
+class Navbar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showLogin: false,
+      showSignup: false,
+    };
+  }
+
+  toggleModal = (modalName) => {
```

Comment:
Don't need extra parenthesis 
```suggestion
  toggleModal = modalName => this.setState(
     prevState => ({ [modalName]: !prevState[modalName] }),
   );
```